src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.il

```diff
@@ -423,3 +437,20 @@
   } // end of method NonVersionableAttribute::.ctor
 
 } // end of class System.Runtime.Versioning.NonVersionableAttribute
+
+#ifdef netcoreapp
+#else
+.class private auto ansi sealed beforefieldinit System.Runtime.CompilerServices.IsReadOnlyAttribute
```

Comment:
@VSadov I see that Roslyn is marking the type with `Microsoft.CodeAnalysis.EmbeddedAttribute`. Is it ok to skip it here?

Reply:
Yes, you don't need to mark as embedded (since this type was not embedded by the compiler).

---
In reply to: [143267591](https://github.com/dotnet/corefx/pull/24479#discussion_r143267591) [](ancestors = 143267591)

Reply:
I think @OmarTawfik already [added the `IsReadOnlyAttribute` type](https://github.com/dotnet/corefx/pull/18637) to coreCLR and coreRT repos. Can you reference it instead of defining it again?

Reply:
This package targets older runtimes too. The local definition is needed for targeting of the older runtimes. `ifdef netcoreapp` avoids the local definition for .NET Core.

Reply:
It is also added to desktop 4.7.1. Not sure if this infrastructure takes that into account:
https://apisof.net/catalog/System.Runtime.CompilerServices.IsReadOnlyAttribute